src/parser/shaman/shared/talents/ElementalBlast.tsx

```diff
@@ -3,25 +3,24 @@ import React from 'react';
 import SPELLS from 'common/SPELLS';
 import SpellIcon from 'common/SpellIcon';
 import { formatPercentage } from 'common/format';
-
 import Analyzer from 'parser/core/Analyzer';
-
 import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+import { ApplyBuffEvent, RemoveBuffEvent } from 'parser/core/Events';
 
-import { ELEMENTAL_BLAST_IDS } from '../../constants';
+import { ELEMENTAL_BLAST_IDS } from '../constants';
 
 class ElementalBlast extends Analyzer {
```

Comment:
did you change the includes for elemental after moving the file?

Reply:
Yep! https://github.com/WoWAnalyzer/WoWAnalyzer/pull/3882/files#diff-0bc09c13471a265c0b46c9cab2279d330a2def5bda5e3a7f2b28de3d86df21a4